main.tf

```diff
@@ -66,6 +66,8 @@ module "vault_cluster" {
   ami_id    = var.ami_id == null ? data.aws_ami.vault_consul.image_id : var.ami_id
   user_data = data.template_file.user_data_vault_cluster.rendered
 
+  iam_instance_profile_arn = var.vault_iam_instance_profile_arn
```

Comment:
There's no reason to chat the code in the root, as that is _example_ code. You only need this update in the modules in the `/modules` folder.